guides/development/hack_on_the_trento.md

```diff
@@ -4,12 +4,24 @@
 
 In order to run the Trento Web application, the following software must be installed:
 
-1. [Elixir](https://elixir-lang.org/)
-2. [Erlang OTP](https://www.erlang.org/)
-3. [Node.js](https://nodejs.org/en/)
+1. [Elixir](https://elixir-lang.org/) - 1.15.7 preferred
+2. [Erlang OTP](https://www.erlang.org/) - 26.1.2 preferred
+3. [Node.js](https://nodejs.org/en/) - 16.16.0 preferred
 4. [Docker](https://docs.docker.com/get-docker/)
 5. [Docker Compose](https://docs.docker.com/compose/install/)
 
+### Additional requirements
+
+Some platforms might not be able to use pre-built versions of some dependencies.
+Therefore, some additional dependencies might be required. This will not effect
+most users and should be referred to when installation issues come up.
+For these dependencies, the distro packaged version will usually be sufficient.
+
+1. [Python3](https://www.python.org/)
+2. [setuptools](https://setuptools.pypa.io/en/latest/index.html)
+3. [gcc](https://gcc.gnu.org/) or [clang](https://clang.llvm.org/)
```

Comment:
We don't really need to add Clang here, do we? We just use GCC according to the Ansible Playbook

Reply:
True